packages/gestalt/src/RadioGroup.js

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+import { type Node } from 'react';
+import Fieldset from './Fieldset.js';
+import Flex from './Flex.js';
+import RadioGroupButton from './RadioGroupButton.js';
+import { RadioGroupContextProvider } from './RadioGroupContext.js';
+
+type Props = {|
+  /**
+   * The individual RadioGroup.RadioButtons for this group
+   *
+   */
+  children: Node,
+  /**
+   * A unique identifier for this RadioGroup. `id` must be specified when an `errorMessage` is added.
+   *
+   */
+  id?: string,
```

Comment:
Should we just make this always required? Even if not strictly necessary, it would eliminate the possibility of forgetting it with errorMessage.